.gitignore

```diff
@@ -1,8 +1,9 @@
 *.pyc
+*.so
+.coverage
+.vagrant
+MANIFEST
```

Comment:
MANIFEST is a generated file that is part of the egg that is uploaded to PyPi.

They are not supposed to be checked in:

http://stackoverflow.com/q/29836718/